guides/common/modules/proc_renewing-a-custom-ssl-certificate-on-smart-proxy.adoc

```diff
@@ -0,0 +1,75 @@
+====
+
+Use this procedure to renew a {customssl} certificate signed by a Certificate Authority for your {SmartProxyServer}.
+The `{foreman-installer}` command, which the `{certs-generate}` command returns, is unique to each {SmartProxyServer}.
+You cannot use the same command on more than one {SmartProxyServer}.
+
+.Prerequisite
+* You must create a new Certificate Signing Request and send it to the Certificate Authority to sign the certificate.
+In return, you will receive the {SmartProxyServer} certificate and CA bundle.
+You must keep the {SmartProxyServer} SSL certificates on the {ProjectServer} by creating a folder and verifying them through the `katello-certs-check` command.
+
+.Procedure
+. On {ProjectServer}, validate the {customssl} certificate input files:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# katello-certs-check \
+-b /root/_{smart-proxy-context}_cert/ca_cert_bundle.pem_ \
+-c /root/_{smart-proxy-context}_cert/new_{smart-proxy-context}_cert.pem_ \
+-k /root/_{smart-proxy-context}_cert/{smart-proxy-context}_cert_key.pem_ \
+----
++
+If the CN has a wildcard value, you must add the `-t {smart-proxy-context}` option to the `katello-certs-check` command.
+. Use the following `{certs-generate}` commands to generate the certificate archive file for your {SmartProxyServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+{certs-generate} --foreman-proxy-fqdn "{smartproxy-example-com}" \
+--certs-tar  "_/root/My_Certificates/{smartproxy-example-com}-certs.tar_" \
+--server-cert "_/root/My_Certificates/new-{smart-proxy-context}_cert.pem_" \
+--server-key "_/root/My_Certificates/{smart-proxy-context}_cert_key.pem_" \
+--server-ca-cert "_/root/My_Certificates/ca_cert_bundle.pem_" \
+--certs-update-server
+----
+. On {ProjectServer}, copy the certificate archive file to your {SmartProxyServer}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# scp /root/_{smart-proxy-context}_cert/{smartproxy-example-com}-certs.tar_
+----
+. Retain a copy of the `{foreman-installer}` command that the `{certs-generate}` command returns for deploying the certificate to your {SmartProxyServer}.
+. Deploy the certificate on your {SmartProxyServer} using the `{foreman-installer}` command returned by the `{certs-generate}` command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario-smartproxy} \
+--foreman-proxy-content-parent-fqdn "{foreman-example-com}" \
+--foreman-proxy-register-in-foreman "true" \
+--foreman-proxy-foreman-base-url \ "https://{foreman-example-com}"
+--foreman-proxy-trusted-hosts \ "{foreman-example-com}" \
+--foreman-proxy-trusted-hosts "{smartproxy-example-com}" \
+--foreman-proxy-oauth-consumer-key "_My_OAuth_Consumer_Key_" \
+--foreman-proxy-oauth-consumer-secret "_My_OAuth_Consumer_Secret_" \
+--foreman-proxy-content-certs-tar "_/root/{smartproxy-example-com}-certs.tar_" \
+--puppet-server-foreman-url "https://{foreman-example-com}" \
```

Comment:
Also, if the Foreman server URL wasn't changed this shouldn't be specified.

It should also be noted that this should only be specified if the user enabled Puppet. If they didn't (default for Katello), this will cause problems.